src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelDiscoveryDataAggregator.cs

```diff
@@ -90,36 +77,32 @@ public ParallelDiscoveryDataAggregator()
     /// Aggregate discovery data
     /// Must be thread-safe as this is expected to be called by parallel managers
     /// </summary>
-    public void Aggregate(
-        long totalTests,
-        bool isAborted,
-        Dictionary<string, HashSet<string>> discoveredExtensions)
+    public void Aggregate(DiscoveryCompleteEventArgs discoveryCompleteEventArgs)
     {
         lock (_dataUpdateSyncObject)
         {
-            IsAborted = IsAborted || isAborted;
+            IsAborted = IsAborted || discoveryCompleteEventArgs.IsAborted;
 
-            if (IsAborted)
-            {
-                // Do not aggregate tests count if test discovery is aborted. It is mandated by
-                // platform that tests count is negative for discovery abort event.
-                // See `DiscoveryCompleteEventArgs`.
-                TotalTests = -1;
-                return;
-            }
-
-            TotalTests += totalTests;
+            // Do not aggregate tests count if test discovery is aborted. It is mandated by
```

Comment:
I doubt anyone is using this "mandated" value, but let's keep it just in case.

Reply:
Text was just moved from another part :)